test/e2e/helpers/registerLogIn.ts

```diff
@@ -7,6 +7,19 @@ export interface CustomBrowserContext extends BrowserContext {
   username?: string
 }
 
+export async function reset() {
```

Comment:
if you have a look at the method `withCleanAliceBobEmail` right below the one you just created, you'll notice we're doing the same thing there, feel free to rename it to reset, BUT see how that one takes an url fir alice and bob? That's because it cannot be hardcoded due to e2e tests. Have a look at `connection-slice-to-bob.spect.ts` to see where we get these urls for. Then also have a look at docker compose and docker compose e2e  to see where  these envs are coming from.